core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -152,10 +152,12 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
 
   /**
    * Compute a histogram using the provided buckets. The buckets are all open
-   * to the right except for the last which is closed
+   * to the right except for the last which is closed.
+   * {{{
    *  e.g. for the array
    *  [1, 10, 20, 50] the buckets are [1, 10) [10, 20) [20, 50]
    *  e.g 1<=x<10 , 10<=x<20, 20<=x<=50
+   * }}}
```

Comment:
`{@code}` is valid in Javadoc; is it saying basically any `{@foo }` syntax is treated as back-tick-quoted by scaladoc? as it happens that's fine here right? can you just disable the warning in your IDE? If it produces the correct output then that's going to be a nicer rendering than breaking out new code blocks.